codex-ui/src/vue/card/Card.vue

```diff
@@ -40,12 +47,30 @@ defineProps<{
   background-color: var(--base--bg-secondary);
 }
 
+.horizontal {
+  width: 100%;
+  flex-direction: row;
+  gap: var(--spacing-l);
+  align-items: center;
+}
+
 .cover {
   width: 100%;
   height: 140px;
   border-radius: var(--radius-m);
   background-color: var(--base--bg-primary);
 }
+
+.footer {
+  width: 100%;
+}
+
+.horizontalCover {
+  width: 150px;
+  height: 100px;
+  border-radius: var(--radius-m);
+  background-color: var(--base--bg-primary);
+}
 .noteTitle {
```

Comment:
please, use nesting, since our CSS modules a setup to leave classed with adding the `.cdx-` prefix. It can be useful for end-user customization — there will be able to override such classes.

So right now it will create many redundant classes like `.cdx-footer`. Let's leave a singe root class `.card` (which will be compiled to the `.cdx-card`) and move all its elements inside it like in BEM

```
.card {
  &--horisontal {
  }
  
  &__cover {
  }
  
  &__title {
  }
  
  ...
}